pagopa/io-app
# chore: [IA-606] Store app versions history

## Short description
This PR adds a persisted section where are the last 10 app version on which the app has been ran

## List of changes proposed in this pull request
- add `installation.appVersionHistory` store section
- change `starupSaga`

## How to test
- run the app with different app versions and check that the store will be right updated
